general/administration/installing.md

```diff
@@ -140,65 +140,52 @@ A community project to deploy Jellyfin on Kubernetes-based platforms exists [at
 
 ### Podman
 
-[Podman](https://podman.io) allows you to run containers as non-root. It's also the offically supported container solution on RHEL and CentOS.
-
-Steps to run Jellyfin using Podman are almost identical to Docker steps:
+[Podman](https://podman.io) allows you to run rootless containers. It's also the officially supported container solution on Fedora Linux and its derivatives such as CentOS Stream and RHEL. Steps to run Jellyfin using Podman are similar to the Docker steps.
 
 1. Install Podman:
 
    ```sh
-   dnf install -y podman
-   ```
-
-2. Download the latest container image:
-
-   ```sh
-   podman pull docker.io/jellyfin/jellyfin
-   ```
-
-3. Create persistent storage for configuration and cache data:
-
-   Either create two persistent volumes:
-
-   ```sh
-   podman volume create jellyfin-config
-   podman volume create jellyfin-cache
+   sudo dnf install -y podman
    ```
 
-   Or create two directories on the host and use bind mounts:
+2. Open the necessary ports in your machine's firewall to permit access to the Jellyfin server from outside the host. This is necessary for rootless Podman because it won't be able to open these automatically. If your distribution uses `firewalld`, the following commands save and load a new firewall rule opening the HTTP port `8096` for TCP connections.  
 
-   ```sh
-   mkdir /path/to/config
-   mkdir /path/to/cache
-   ```
+    ```sh
+    sudo firewall-cmd --add-port=8096/tcp --permanent
+    sudo firewall-cmd --reload
+    ```
```

Comment:
IMHO opening the firewall should be one of the last steps after initially setting up the container.
Accessint it via localhost should be possible without opening the firewall, right?

Reply:
You can definitely access the Jellyfin container on localhost without this. I wanted to add this step since it can be confusing if you're used to Docker / rootfull Podman where this is done for you automatically. I'll go ahead and move it to a later step.

Reply:
I've placed the firewall step last and tried to word it such that it emphasizes opening firewall ports is only necessary if you wish to access the Jellyfin container from outside the host.